tutorials/08_restriction_estimator.ipynb

```diff
@@ -0,0 +1,382 @@
+      "text/plain": [
+       "<Figure size 216x216 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "restriction_estimator.append_simulations(new_theta, new_x)  # Gather the new simulations in the `restriction_estimator`.\n",
+    "all_theta, all_x, _ = restriction_estimator.get_simulations()  # Get all simulations run so far.\n",
+    "\n",
+    "inference = SNPE(prior=prior)\n",
+    "density_estimator = inference.append_simulations(all_theta, all_x).train()\n",
+    "posterior = inference.build_posterior()\n",
+    "\n",
+    "posterior_samples = posterior.sample((10_000,), x=torch.ones(2))\n",
+    "_ = pairplot(posterior_samples, limits=[[-2, 2], [-2, 2]], fig_size=(3, 3))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Further options for tuning the algorithm\n",
+    "\n",
+    "- the whole procedure can be repeated many times (see the loop shown in \"Main syntax\" in this tutorial)  \n",
+    "- the classifier is trained to be relatively conservative, i.e. it will try to be very sure that a specific parameter set can indeed not produce `valid` simulation outputs. If you are ok with the restricted prior potentially ignoring a small fraction of parameter sets that might have produced `valid` data, you can use `restriction_estimator.restrict_prior(allowed_false_negatives=...)`. The argument `allowed_false_negatives` sets the fraction of potentially ignored parameter sets. A higher value will lead to more `valid` simulations.  \n",
+    "- By default, the algorithm considers simulations that have at least one `NaN` of `inf` as `invalid`. You can specify custom criterions with `RestrictionEstimator(decision_criterion=...)`\n",
+    "- For differencese between the `RestrictionEstimator` and [multi-round inference](https://www.mackelab.org/sbi/tutorial/03_multiround_inference/), please see Deistler et al. 2021 (in preparation)."
```

Comment:
You often refer to Deistler et al 2021 in preparation. Will there be a preprint soon so that people can look it up? :)